pytroll_collectors/tests/test_scisys.py

```diff
@@ -216,14 +217,51 @@ def test_reception(self):
         os.remove(filename)
 
 
-def suite():
-    """Test suite for test_scisys."""
-    loader = unittest.TestLoader()
-    mysuite = unittest.TestSuite()
-    mysuite.addTest(loader.loadTestsFromTestCase(ScisysReceiverTest))
+def test_get_subject_from_msg2send_viirs():
+    """Test the get the subject from the message being send - viirs data."""
+    msg_rec = MessageReceiver("nimbus")
 
-    return mysuite
+    filename = os.path.join('/tmp', VIIRS['uid'])
+    touch(filename)
+    string = TwoMetMessage(INPUT_DISPATCH_VIIRS)
+    to_send = msg_rec.receive(string)
+    os.remove(filename)
 
+    topic_postfix = None
+    station = 'nrk'
+    environment = 'dev'
+    subject = _get_subject_from_msg2send(to_send, station, environment, topic_postfix)
 
-if __name__ == '__main__':
-    unittest.main()
+    assert subject == "/viirs/RDR/0/nrk/dev/polar/direct_readout"
+
+    station = 'nrk'
+    environment = 'dev'
+    topic_postfix = 'my_topic'
+    subject = _get_subject_from_msg2send(to_send, station, environment, topic_postfix)
+
+    assert subject == "/viirs/RDR/0/my_topic"
```

Comment:
This is another test case, so should be separate to the previous one.

Reply:
Done